<!DOCTYPE HTML>
<!--
	Read Only by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Samuel Barnett</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
			<section id="header">
				<header>
					<span class="image avatar"><img src="images/SamPenguins.jpg" alt="Visiting the Magellanic penguins on Isla Magdalena in Chile" /></span>
					<h1 id="logo"><a href="#">Samuel Barnett</a></h1>
					<p>Postdoctoral Associate<br />
					Microbial Ecologist</p>
				</header>
				<nav id="nav">
					<ul>
						<li><a href="#one" class="active">About Me</a></li>
						<li><a href="#two">Research</a></li>
						<li><a href="#three">Teaching and Outreach</a></li>
						<li><a href="#four">Publications</a></li>
						<li><a href="#five">Contact</a></li>
					</ul>
				</nav>
				<footer>
					<ul class="icons">
						<li><a href="https://twitter.com/SamBarnett15" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
						<li><a href="https://github.com/seb369" class="icon brands fa-github"><span class="label">Github</span></a></li>
						<li><a href="https://www.linkedin.com/in/samuel-barnett-8a177394" class="icon brands fa-linkedin"><span class="label">LinkedIn</span></a></li>
					</ul>
				</footer>
			</section>

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Main -->
					<div id="main">

						<!-- One -->
							<section id="one">
								<div class="image main" data-position="center">
									<img src="images/Flowers.jpeg" alt="Field of flowers near Half Moon Bay, CA" />
								</div>
								<div class="container">
									<header class="major">
										<h2>Samuel Barnett</h2>
										<p>Postdoctoral Associate in the <a href="https://ashley17061.wixsite.com/shadelab">Shade Lab</a> at<br />Michigan State University</p>
									</header>
									<div class="row">
										<div class="col-8">
											<p>I am a microbial ecologist interested in the complex microbial communities in the soils beneath our feet and the essential roles these microbes play in our lives. I am particularly intrigued by how the highly diverse and complex microbial communities we observe in soils and other environments assemble across natural and disturbed ecosystems and how these processes may influence community wide functions.<br /><br />I recently started a Postdoc with <a href="https://mmg.natsci.msu.edu/people/faculty/shade-ashley/">Dr. Ashley Shade</a> in the <a href="https://mmg.natsci.msu.edu">Department of Microbiology and Molecular Genetics</a> at Michigan State University. My current work looks at bacterial community resilience to long-term heat press disturbance in soils above the <a href="https://ashley17061.wixsite.com/shadelab/centralia">Centralia, PA coal seam fires</a>.<br /><br />I did my Ph.D. in the <a href="https://micro.cornell.edu">Graduate Field of Microbiology</a> at Cornell University with <a href="https://cals.cornell.edu/daniel-h-buckley">Dr. Daniel Buckley</a>. My graduate work focused on how the structure of soil microbial communities varies across habitats, particularly land-use types, and how this variation affects microbial activities in <a href="https://blogs.cornell.edu/buckley/">soil carbon cycling</a>.</p>
										</div>
										<div class="col-4">
											<h4 style="text-align:center">Education</h4>
											<p><b>Ph.D. Microbiology</b>&emsp;&emsp;&ensp;&nbsp;2021<br />&emsp;Cornell University<br /><b>B.Sc. Biological Sciences</b>&ensp;2015 <br />&emsp;University of Maryland<br /><b>B.Sc. Mathematics</b>&emsp;&emsp;&emsp;2015 <br />&emsp;University of Maryland</p>
										</div>
									</div>
								</div>
							</section>

						<!-- Two -->
							<section id="two">
								<div class="container">
									<h3><u>Research Interests</u></h3>
									<div class="features">
										<article>
											<div class="image"><img src="images/Ithaca_field.jpeg" alt="Field site in Ithaca, NY" /></div>
											<div class="inner">
												<h4>Bacterial community assembly</h4>
												<p>The study of community assembly aims to describe the distinct ecological processes (i.e., selection, dispersal, and ecological drift) that regulate community structure and composition. The processes underlying how a bacterial community is assembled can have profound impacts on its functioning, especially under a changing climate and other environmental conditions. As part of my graduate work, I examined how bacterial community assembly in soil varied across distinct land-use regimes. This work has been published in <a href="https://doi.org/10.1093/femsec/fiz194">FEMS Microbiology Ecology</a>.</p>
											</div>
										</article>
										<article>
											<div class="image"><img src="images/Ccyling.png" alt="C-cycling diagram" /></div>
											<div class="inner">
												<h4>Microbially mediated carbon cycling</h4>
												<p>Microbes are essential components of the global carbon cycle. In soils, microbes are significantly involved in biomass turnover, creation of soil organic matter, and loss of carbon through soil respiration. Exploring the diversity and ecology of the microbes active in these processes is important in understanding the dynamics of carbon flux and the fate of soil carbon. During my graduate studies, I examined how aspects of the environment and carbon source influenced the activity of soil bacteria in turnover of organic matter. Studies on this work have been published in <a href="https://doi.org/10.1073/pnas.2115292118">The Proceedings of the National Academy of Sciences</a> or are available on <a href="https://doi.org/10.1101/2022.02.16.480692">bioRxiv</a>.</p>
											</div>
										</article>
										<article>
											<div class="image"><img src="images/Fluorescent_bacteria.jpg" alt="Pictures of some random fluorescent bacteria I took" /></div>
											<div class="inner">
												<h4>Bacterial life history strategies</h4>
												<p>Life history strategies explain how an organism allocates energy and resources to growth and maintenance. Trade-offs in traits across life history strategies can define a microbe's activity in environmental processes, notably carbon cycling. As part of my graduate work, continuing into my postdoc, I studied the genomics of bacterial life history strategies in soil.</p>
											</div>
										</article>
										<article>
											<div class="image"><img src="images/SIP.png" alt="DNA-SIP diagram" /></div>
											<div class="inner">
												<h4>DNA stable isotope probing methodologies</h4>
												<p>For much of my graduate work, I utilized DNA stable isotope probing (DNA-SIP) methodologies involving both amplicon and metagenome high throughput sequencing. As part of DNA-SIP methods development, I created a python-based toolkit to simulate metagenomic-SIP datasets called <a href="https://github.com/seb369/MetaSIPSim"> MetaSIPSim</a>, which was published in <a href="https://doi.org/10.1186/s12859-020-3372-6">BMC Bioinformatics</a>.</p>
											</div>
										</article>
									</div>
								</div>
							</section>

						<!-- Three -->
							<section id="three">
								<div class="container">
									<h3><u>Teaching and Outreach</u></h3>
									<div class="features">
										<article>
											<h4>Teaching experience</h4>
											<p>I greatly value my role in preparing students for futures in academia, industry, medicine, policy, and communication. During my postdoc, graduate, and undergraduate careers, I have been fortunate enough to participate in a number of instructor and teaching assistant opportunities in lecture, discussion, and laboratory based courses. Through these assistantships I have learned teaching methodologies and philosophies from some dedicated educators. In addition to these teaching experiences, I have taken formal coursework and workshops on teaching through Cornell University's Center for Teaching Innovation including <i>Teaching in Higher Education</i> and <i>Teaching & Learning in the Diverse Classroom</i>. My personal teaching philosophy emphasizes developing students' critical thinking skills and getting students to apply core course concepts beyond in-class examples. I also strive to create an inclusive classroom environment, where all students feel safe and encouraged to participate in the learning process. I am currently the instructor for a topics in microbiology section at Michigan State (MMG991) focused on statistical and computation methods in microbiology and microbial ecology.</p>
										</article>
										<article>
											<h4>Outreach experience</h4>
											<p>Engaging with the community is an essential part of open science. I feel that we must give back to the communities who support our research. I am particularly passionate about working with K-12 students, introducing them to the microscopic worlds around them that are so important to their lives. To this end, I have been involved in a number of outreach activities.</p>
											<div class="image">
												<figure>
													<img src="images/SIPSBioBlitz_2017.jpeg" alt="Team Microbe booth at the 2017 SIPS BioBlitz" />
													<figcaption>Our booth at the 2017 SIPS BioBlitz</figcaption>
												</figure>
												<br/>
												<figure>
													<img src="images/USASEF_2018.jpeg" alt="Team Microbe at the 2018 USA Science and engineering festival in Washington, DC" />
													<figcaption>Team Microbe at the 2018 USASEF</figcaption>
												</figure>
											</div>
											<div class="inner">
												<ul>
													<li>I was a member of the <i>Microbiome Sciences at Cornell University Team</i> at the 2017 Cayuga Nature Center BioBlitz and the 2018 USA Science and Engineering Festival in Washington, DC.</li>
													<li>With a fellow microbiology graduate student as part of Cornell University's GRASSHOPR Program, I developed and taught lessons focusing on human and environmental microbiomes for 6th grade students at Lansing Middle School in Lansing, NY.</li>
													<li>Throughout most of my time in graduate school I was a volunteer bluecoat at the Ithaca Sciencenter, helping kids engage with science through hands-on activities such as a tide pool touch tank.</li>
												</ul>
										</article>
									</div>
								</div>
							</section>

						<!-- Four -->
							<section id="four">
								<div class="container">
									<h3><u><a href="https://scholar.google.com/citations?user=1d9AORUAAAAJ&hl=en">Publications</a></u></h3>
									<div class="features">
										<ol>
											
<li><i>(PREPRINT)</i><b>Barnett SE</b>, Egan R, Foster B, Eloe-Fadrosh EA, Buckley DH. Genomic features predict bacterial life history strategies in soil, as identified by metagenomic stable isotope probing. bioRxiv 2022; DOI: <a href="https://doi.org/10.1101/2022.09.09.507310">10.1101/2022.09.09.507310</a></li>
<li>Wilhelm RC, <b>Barnett SE</b>, Swenson T, Youngblut ND, Koechli CN, Bowen BP, Northen T, Buckley DH (2022). Tracing carbon metabolism with stable isotope metabolomics reveals the legacy of diverse carbon sources in soil. Applied and Environmental Microbiology, e00839-22. DOI: <a href="https://doi.org/10.1128/aem.00839-22">10.1128/aem.00839-22</a></li>
<li><b>Barnett SE</b>, Youngblut ND, Buckley DH (2022). Bacterial community dynamics explain carbon mineralization and assimilation in soils of different land-use history. Environmental Microbiology, 1-18. DOI: <a href="https://doi.org/10.1111/1462-2920.16146">10.1111/1462-2920.16146</a></li>
<li><b>Barnett SE</b>*, Youngblut ND*, Koechli CN, Buckley DH (2021). Multi-substrate DNA stable isotope probing reveals guild structure in bacterially mediated soil carbon cycling. PNAS, 118:e2115292118. DOI: <a href="https://doi.org/10.1073/pnas.2115292118">10.1073/pnas.2115292118</a>&emsp;&emsp;*co-first authors</li>
											<li>Yavitt JB, Roco CA, Debenport S, <b>Barnett SE</b>, Shapleigh JP (2020). Community Organization and Metagenomics of Bacterial Assemblages Across Local Scale pH Gradients in Northern Forest Soils. Microbial Ecology. DOI: <a href="https://doi.org/10.1007/s00248-020-01613-7">10.1007/s00248-020-01613-7</a></li>
											<li><b>Barnett SE</b>, Cala AR, Hansen JL, Crawford J, Viands DR, Smart LB, Smart CD, Buckley DH (2020). Evaluating the microbiome of hemp. Phytobiomes Journal, 4:351–363. DOI: <a href="https://doi.org/10.1094/PBIOMES-06-20-0046-R">10.1094/PBIOMES-06-20-0046-R</a></li>
											<li><b>Barnett SE</b>, Buckley DH (2020). Simulating metagenomic stable isotope probing datasets with MetaSIPSim. BMC Bioinformatics, 21:37. DOI: <a href="https://doi.org/10.1186/s12859-020-3372-6">10.1186/s12859-020-3372-6</a></li>
											<li>Kapili BJ, <b>Barnett SE</b>, Buckley DH, Dekas AE (2020) Evidence for phylogenetically and catabolically diverse active diazotrophs in deep-sea sediment. ISME Journal, 14:971–983. DOI: <a href="https://doi.org/10.1038/s41396-019-0584-8">10.1038/s41396-019-0584-8</a></li>
											<li><b>Barnett SE</b>, Youngblut ND, Buckley DH (2020) Soil characteristics and land-use drive bacterial community assembly patterns. FEMS Microbiology Ecology, 96:fiz194. DOI: <a href="https://doi.org/10.1093/femsec/fiz194">10.1093/femsec/fiz194</a></li>
											<li><b>Barnett SE</b>, Youngblut ND, Buckley DH (2019). Data Analysis for DNA Stable Isotope Probing Experiments Using Multiple Window High-Resolution SIP. In: Stable Isotope Probing. Dumont M, Hernández García M (ed.). Methods in Molecular Biology, vol 2046. Humana, New York. DOI: <a href="https://doi.org/10.1007/978-1-4939-9721-3_9">10.1007/978-1-4939-9721-3_9</a></li>
											<li>Muletz-Wolz CR, <b>Barnett SE</b>, DiRenzo GV, Zamudio KR, Toledo LF, James TY, Lips KR (2019). Diverse genotypes of the amphibian killing fungus produce distinct phenotypes through plastic responses to temperature. Journal of Evolutionary Biology, 32:287–298. DOI: <a href="https://doi.org/10.1111/jeb.13413">10.1111/jeb.13413</a></li>
											<li>Youngblut ND, <b>Barnett SE</b>, Buckley DH (2018). SIPSim: A Modeling Toolkit to Predict Accuracy and Aid Design of DNA-SIP Experiments. Frontiers in Microbiology, 9:570. DOI: <a href="https://doi.org/10.3389/fmicb.2018.00570">10.3389/fmicb.2018.00570</a></li>
											<li>Youngblut ND, <b>Barnett SE</b>, Buckley DH (2018). HTSSIP: An R package for analysis of high throughput sequencing data from nucleic acid stable isotope probing (SIP) experiments. PLoS ONE, 13:e0189616. DOI: <a href="https://doi.org/10.1371/journal.pone.0189616">10.1371/journal.pone.0189616</a></li>
											<li>Muletz-Wolz CR, Almario JG, <b>Barnett SE</b>, DiRenzo GV, Martel A, Pasmans F, Zamudio KR, Toledo LF, Lips KR (2017). Inhibition of Fungal Pathogens across Genotypes and Temperatures by Amphibian Skin Bacteria. Frontiers in Microbiology, 8:1551. DOI: <a href="https://doi.org/10.3389/fmicb.2017.01551">10.3389/fmicb.2017.01551</a></li>
										</ol>
									</div>
								</div>
							</section>

						<!-- Five -->
							<section id="five">
								<div class="container">
									<h3><u>Contact Me</u></h3>
									<p>If you are interested in my research or have any questions, please feel free to contact me. I can also be reached through <a href="https://twitter.com/SamBarnett15">Twitter</a> or <a href="https://www.linkedin.com/in/samuel-barnett-8a177394">LinkedIn</a>.</p>
									<form method="POST" action="https://formspree.io/f/xqkgrzjn">
										<label>Name:<input type="text" name="name" required/></label>
										<label>Email:<input type="email" name="_replyto" required/></label>
										<label>Message:<textarea name="message" required/></textarea></label>
										<button type="submit">Send</button>
										<button type="reset">Reset form</button>
									</form>
								</div>
							</section>

					</div>

				<!-- Footer -->
					<section id="footer">
						<div class="container">
							<ul class="copyright">
								<li>&copy; Samuel Barnett 2022. All rights reserved.</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</section>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>